include/mega/megaclient.h

```diff
@@ -1534,7 +1534,10 @@ class MEGA_API MegaClient
     void acknowledgeuseralerts();
 
     // manage overquota errors
-    void activateoverquota(dstime timeleft);
+    void activateoverquota(dstime timeleft, handle targetHandle = UNDEF);
```

Comment:
Perhaps it's good to document in which cases `targetHandle == UNDEF`:
- The API returns -17 due to bandwidth overquota for any API command (other than `u` and `p`)
- The api returns -17 due to storage overquota for any API command (other than `u` and `p`)